CHANGELOG

```diff
@@ -13,13 +13,17 @@ The rules for CHANGELOG file:
 
 Unreleased
 ----------
+- Add ``_BaseKPCov`` class (#254)
+- Add ``KernelPCovC`` class that inherits shared functionality from ``_BaseKPCov`` (#254)
+- Add ``KernelPCovC`` testing suite and examples (#254)
+- Modify ``KernelPCovR`` to inherit shared functionality from ``_BaseKPCov`` (#254)
```

Comment:
I think one line saying adding ``KernelPCovC`` and what it does should be enough. Usually users don't care about base classes and tests.